src/app/common/app.helpers.ts

```diff
@@ -147,3 +147,37 @@ export function addMarkdownRunButton(sqlQueries: RegExpMatchArray | null, path:
         });
     }
 }
+
+export function operationColumnMapper(value: string | number): string {
+    if (typeof value === "number") {
+        switch (value) {
+            case 0:
+                return "+A";
+            case 1:
+                return "-R";
+            case 2:
+                return "-C";
+            case 3:
+                return "+C";
+            /* istanbul ignore next */
+            default:
+                throw new Error("Unknown operation type");
+        }
+    } else return value;
+}
```

Comment:
Let's have a better place for this. It is not a general-purpose application wide utility helper. It is specific to presenting data.